jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/util/NotificationPermissionUtils.java

```diff
@@ -0,0 +1,68 @@
+package com.tmobile.cso.pacman.inventory.util;
+
+import com.tmobile.cso.pacman.inventory.config.ConfigUtil;
+import com.tmobile.cso.pacman.inventory.vo.ErrorVH;
+import com.tmobile.cso.pacman.inventory.vo.PermissionNotificationRequest;
+import com.tmobile.cso.pacman.inventory.vo.PermissionVH;
+import com.tmobile.pacman.commons.dto.NotificationBaseRequest;
+import com.tmobile.pacman.commons.utils.Constants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.Gson;
+
+import static com.tmobile.pacman.commons.PacmanSdkConstants.POLICY_NAME;
+
+public class NotificationPermissionUtils {
```

Comment:
Can you move this file to batch-commons and reuse it instead of creating it twice?

Reply:
moved